xbmc/interfaces/json-rpc/JSONServiceDescription.cpp

```diff
@@ -1154,9 +1154,9 @@ void JSONSchemaTypeDefinition::Set(const JSONSchemaTypeDefinitionPtr typeDefinit
   referencedTypeSet = true;
 }
 
-JSONSchemaTypeDefinition::CJsonSchemaPropertiesMap::CJsonSchemaPropertiesMap()
+JSONSchemaTypeDefinition::CJsonSchemaPropertiesMap::CJsonSchemaPropertiesMap():
+   m_propertiesmap(std::map<std::string, JSONSchemaTypeDefinitionPtr>())
```

Comment:
Does this really need to be so explicit? Wouldn't `m_propertiesmap()` be enough?
Personally I'd also prefer it if either the colon would be on the new line followed by a space or at least add a space between the `)` and the colon for easier readability.